app/views/content/blog/11-top-tips-for-interview-success.md

```diff
@@ -17,10 +17,11 @@ keywords:
 tags:
   - becoming a teacher
   - applications
-closing_paragraph: enriching-the-lives-of-young-people
 ---
 
-$interview$
+Caroline Downing is the Primary Programme Leader at the [Bedfordshire Schools' Training Partnership](https://bedsscitt.org.uk). She interviews applicants and assesses their potential to become successful classroom professionals.
```

Comment:
@gemmadallmandfe moved this info to the top - felt better situated here?

Reply:
Agree now that we have lost the image on this page